fixtures/ios_app_with_frameworks/App/Project.swift

```diff
@@ -3,14 +3,15 @@ import ProjectDescription
 let settings = Settings(base: [
     "HEADER_SEARCH_PATHS": "path/to/lib/include",
 ])
+
 let project = Project(name: "MainApp",
                       settings: settings,
                       targets: [
                           Target(name: "App",
                                  platform: .iOS,
                                  product: .app,
                                  bundleId: "io.tuist.App",
-                                 infoPlist: "Config/App-Info.plist",
+                                 infoPlist: .extendingDefault(with: [:]),
```

Comment:
is it worth adding a custom value?

Reply:
That's already being tested with unit tests, so I don't think it adds much value.